runbotics/runbotics-orchestrator-ui/src/src-app/views/process/ProcessConfigureView/ProcessCredentials/ActionCredential.tsx

```diff
@@ -87,7 +87,7 @@ const ActionCredential: FunctionComponent<ActionCredentialProps> = ({
                 </CredentialDetails>
                 <CredentialDelete $isPrimary={isPrimary}>
                     <DeleteOutlineIcon
-                        {...(!isPrimary && { onClick: () => handleDeleteDialog(credentialId) })}
+                        onClick={() => handleDeleteDialog(credentialId)}
```

Comment:
Is it intentional?

Reply:
was necessary for the demo, and I think Jan told me it's strange the primary credential cannot be removed. But I'll restore it and we will decide together.